src/cobalt.rs

```diff
@@ -133,15 +133,40 @@ fn generate_doc(
 ) -> Result<()> {
     // Everything done with `globals` is terrible for performance.  liquid#95 allows us to
     // improve this.
-    let mut globals: liquid::value::Object = vec![
-        (
-            "site".into(),
-            liquid::value::Value::Object(context.site.clone()),
-        ),
-        global_collection,
-    ]
-    .into_iter()
-    .collect();
+    let mut globals: liquid::value::Object =
+        if doc.front.pagination.is_some() || doc.front.pagination_compat {
```

Comment:
I feel like this needs to be an `&&` and not an `||`.  If someone has a paginator configured and doesn't have compat set, then we shouldn't warn people / add `collections`

Reply:
Duh, you're right, I didn't do any test yet, it was to show you how I was going to do the aliasing. Good catch, though!